File: spec/code_samples/JavaScript/estimate_coin_sell/get.js
import {Minter, SendTxParams, API_TYPE_NODE} from "minter-js-sdk";

const minterSDK = new Minter({apiType: API_TYPE_NODE, baseURL: 'https://minter-node-1.testnet.minter.network:8841'});

minterSDK.estimateCoinSell({
        coinToSell: 'MNT',
        valueToSell: 1,
        coinToBuy: 'ASDCOIN',
    })
    .then((estimateResult) => {
        console.log(estimateResult.will_get);
        console.log(estimateResult.commission);
    })
    .catch((error) => {
        const errorMessage = error.response.data.error.message;
        console.log(errorMessage);
    });

File: spec/code_samples/Swift/estimate_coin_sell/get.swift
import MinterCore

MinterCoreSDK.initialize(urlString: "https://minter-node-1.testnet.minter.network:8841/")

let manager = TransactionManager.default

manager.estimateCoinSell(from: "MNT", to: "VALIDATOR", amount: Decimal(string: "10000000000")!, completion: { (willPay, commission, error) in
	print(error)
	print(willPay)
	print(commission)
})
